docs/af5-getting-started/modules/ROOT/pages/project-structure.adoc

```diff
@@ -42,7 +42,7 @@ Please ensure you have the following properties and dependencies in your `pom.xm
 
     <properties>
         <assertj.version>3.27.3</assertj.version>
-        <axon.version>5.0.0-RC1</axon.version>
+        <axon.version>5.0.1</axon.version>
```

Comment:
Ow, if you have these doc pointers...does that mean you based this PR from `axon-5.0.x`? Perfectly fine if so, but that means we'd need to backport `axon-5.0.x` to `main` to ensure we don't review this, even though it has nothing to do with this PR.

Reply:
Or, we point this PR to `axon-5.0.x` instead of `main`.

Reply:
Updated the branches - all `adoc` related stuff is now gone.